runtime/lua/vim/lsp/rpc.lua

```diff
@@ -11,7 +11,7 @@ local validate, schedule, schedule_wrap = vim.validate, vim.schedule, vim.schedu
 ---@param data (table) Data to encode
 ---@returns (string) Encoded object
 local function json_encode(data)
-  local status, result = pcall(vim.fn.json_encode, data)
+  local status, result = pcall(vim.json.encode, data)
```

Comment:
do we still need these wrappers? also the TODO on line 7 could be removed I think